src/services.ts

```diff
@@ -350,3 +350,283 @@ export type {
   ServicesAccessor
 }
 export { ConfigurationTarget } from 'vs/platform/configuration/common/configuration'
+export { IRemoteExplorerService } from 'vs/workbench/services/remote/common/remoteExplorerService.service'
```

Comment:
Home come there are so many new exports?

Reply:
Did you check the commit message?